tools/cloud-build/provision/trigger-schedule/main.tf

```diff
@@ -19,10 +19,9 @@ resource "google_cloud_scheduler_job" "schedule" {
 
   attempt_deadline = "180s"
   retry_config {
-    max_backoff_duration = "0s"
-    max_doublings        = 5
-    max_retry_duration   = "3600s"
-    min_backoff_duration = "1m"
+    max_backoff_duration = "1200s"
+    max_doublings        = 2
+    min_backoff_duration = "300s"
```

Comment:
So my understanding of time between tests
1st - 5m - 2nd - 10m - 3rd - 20m - 4th - 20m - 5th

close if you agree.

Reply:
Yes, but writing down a bit differently to include the initial failure.

FAIL -> 5m -> Retry 1 (FAIL) -> 10m -> Retry 2 (FAIL) -> 20m -> Retry 3 (FAIL) -> 20m -> Retry 4 (FAIL) -> 20m -> Retry 5 (FAIL) -> Done